From 28da6196a27de951d6143d4f2765d1f2976f0d39 Mon Sep 17 00:00:00 2001
From: Mark Wielaard <mark@klomp.org>
Date: Wed, 20 Apr 2022 00:31:01 +0200
Subject: [PATCH] Define STDERR_FILENO for BZ_LCCWIN32

STDERR_FILENO is *nix specific and is not defined under MSVC.
So define it using _fileno(stderr).

Suggested-by: Dmitry Tsarevich <dimhotepus@gmail.com>
---
 bzip2.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/bzip2.c b/bzip2.c
index d1f2fa8..1538faf 100644
--- a/bzip2.c
+++ b/bzip2.c
@@ -150,6 +150,8 @@
          ERROR_IF_MINUS_ONE ( retVal );               \
       } while ( 0 )
 
+#   define STDERR_FILENO _fileno(stderr)
+
 #endif /* BZ_LCCWIN32 */
 
 
-- 
2.31.1

